src/client/eppo-client.spec.ts

```diff
@@ -613,13 +625,13 @@ describe('EppoClient E2E test', () => {
       client.useNonExpiringInMemoryAssignmentCache();
 
       // original configuration version
-      storage.setEntries({ [flagKey]: mockFlag });
+      setUnobfuscatedFlagEntries({ [flagKey]: mockFlag });
```

Comment:
why not `awaiting` some of these?

Reply:
That's a good catch. These tests have still worked because the underlying Flag Config Storage is a memory-only store, so the actual "writing" of entries is happening sync when the method is called.
Worth it to fix it now before it causes weird problems in the future.